site/content/docs/0.0/layout/breakpoints.md

```diff
@@ -11,4 +11,188 @@ aliases:
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Core concepts
+
+- **Breakpoints are the building blocks of responsive design.** Use them to control when your layout can be adapted at a particular viewport or device size.
+
+- **Use media queries to architect your CSS by breakpoint.** Media queries are a feature of CSS that allow you to conditionally apply styles based on a set of browser and operating system parameters. We most commonly use `min-width` in our media queries.
+
+- **Mobile first, responsive design is the goal.** OUDS Web's CSS aims to apply the bare minimum of styles to make a layout work at the smallest breakpoint, and then layers on styles to adjust that design for larger devices. This optimizes your CSS, improves rendering time, and provides a great experience for your visitors.
+
+## Available breakpoints
+
+OUDS Web includes eight default breakpoints, sometimes referred to as _grid tiers_, for building responsively. These breakpoints can be customized if you're using our source Sass files.
+
+{{< bs-table "table" >}}
+| Breakpoint | Class infix | Dimensions |
+| --- | --- | --- |
+| 2X-small | <em>None</em> |&lt;390px |
+| X-small | `xs` | &ge;390px |
+| Small | `sm` | &ge;480px |
+| Medium | `md` | &ge;736px |
+| Large | `lg` | &ge;1024px |
+| X-large | `xl` | &ge;1320px |
+| 2X-large | `2xl` | &ge;1640px |
+| 3X-large | `3xl` | &ge;1880px |
+{{< /bs-table >}}
+
+Breakpoints are representative of a subset of common device sizes and viewport dimensions—they don't specifically target every use case or device. Instead, the ranges provide a strong and consistent foundation to build on for nearly any device.
```

Comment:
Missing sentence from Bootstrap, do we add it?

> Each breakpoint was chosen to comfortably hold containers whose widths are multiples of 12.

Reply:
It's on purpose because our containers widths are not multiples of 12 (and they weren't neither in Boosted by the way)